server/datastore/mysql/migrations/tables/20231024105026_AddWindowsMDMTables.go

```diff
@@ -0,0 +1,111 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20231024105026, Down_20231024105026)
+}
+
+func Up_20231024105026(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+-- This table contains the commands, which may target multiple devices.
+CREATE TABLE windows_mdm_commands (
+	-- managed and generated by Fleet, and used as CmdID in the MSMDM messages.
+	command_uuid     VARCHAR(127) NOT NULL,
+	-- the raw XML of the command
+	raw_command      MEDIUMTEXT NOT NULL,
+	-- whether the command was generated from the system or from a user
+	system_origin    TINYINT(1) NOT NULL DEFAULT FALSE,
+	-- the target OMADM URI for the command
+	target_loc_uri   VARCHAR(255) NOT NULL,
+
+	created_at       TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+	updated_at       TIMESTAMP DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+
+	PRIMARY KEY (command_uuid)
+)`)
+	if err != nil {
+		return fmt.Errorf("failed to create windows_mdm_commands table: %w", err)
+	}
+
+	_, err = tx.Exec(`
+CREATE TABLE windows_mdm_command_queue (
+	host_uuid    VARCHAR(255) NOT NULL,
+	command_uuid VARCHAR(127) NOT NULL,
+
+	-- whether the command should be processed or not (can be deactivated to be ignored)
+	active       BOOLEAN NOT NULL DEFAULT 1,
+
+	created_at   TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+	updated_at   TIMESTAMP DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+
+	PRIMARY KEY (host_uuid, command_uuid),
+
+	-- NOTE: no foreign key added for host_uuid, we don't add FKs to the hosts table.
```

Comment:
This means we will have to delete those rows manually as part of the delete host processing.